receiver/kafkareceiver/zipkin_unmarshaler_test.go

```diff
@@ -113,13 +113,13 @@ func TestUnmarshalZipkinThrift_error(t *testing.T) {
 func TestUnmarshalZipkinJSON_error(t *testing.T) {
 	p := zipkinJSONSpanUnmarshaler{}
 	got, err := p.Unmarshal([]byte("+$%"))
-	assert.Equal(t, pdata.NewTraces(), got)
+	assert.Equal(t, pdata.Traces{}, got)
```

Comment:
No need to compare that is invalid, if error we should not use the pdata result. Probably remove this line.